docs/how-to-guides/openstack-swift.md

```diff
@@ -0,0 +1,42 @@
+# Content-cache with Openstack/Swift storage
+
+Sometimes it is desirable to cache swift storage objects inside the charm for faster processing and reducing the number of requests to the [swift](https://docs.openstack.org/swift/latest/) server. It is also worth mentioning that this allows you to have a different hostname than the one from the Swift service.
```

Comment:
When writing documentation, try to keep it as simple as possible. I'd go with "This will also allow " instead of "It is also worth mentioning that this allows". Can you double check if anything else can be simplified?